css/main.css

```diff
@@ -527,6 +531,34 @@ button.icon.edit:after
   background-image: url(/css/img/edit.svg);
 }
 
+.tableList > li > div .icon.whitelist::after
+{
+  background-image: url(/css/img/check-mark-circle.svg);
+  background-size: 105%;
```

Comment:
Was there a reason you used 105% ?
This causes the icon on my Mac to be cut:
<img width="300" alt="screen shot 2018-12-16 at 5 04 52 pm" src="https://user-images.githubusercontent.com/716969/50055877-db525f00-0154-11e9-8f76-89ed8b5c8057.png">

Reply:
100% looks fine on my screen.

Reply:
yeah i wanted to make the icon a little bit bigger, but now you mention it it does look a bit cut
will set it back to 100